### ISSUE TITLE
Add support for GitLab

### ISSUE THREAD

**Issue by tallesl** [rgrove/rawgit#135]
I would love to see it supporting [gitlab.com](https://gitlab.com)...

**Comment by rgrove** [comment]
Dupe of #49.

**Comment by tallesl** [comment]
> ["This issue is about Bitbucket. Feel free to open a new issue for Gitlab, but please don't hijack this one. Thanks!"](https://github.com/rgrove/rawgit/issues/49#issuecomment-71405117)

**Comment by rgrove** [comment]
Sorry, you're right. When I saw this, I thought, "I'm sure there was another issue about this," then found that one, read the first two comments, and assumed that was where I had already discussed GitLab support. I should've kept reading.

Anyway, my official answer on this is that I have no plans to add support for additional services. RawGit's traffic has already grown to a point where it's getting tricky to manage given the small amount of time I have to devote to it, and I don't want to increase that burden.
